src/core_modules/capture-core/components/DataEntry/actions/dataEntryLoadEdit.actions_old.js

```diff
@@ -8,7 +8,7 @@ import { convertValue } from '../../../converters/clientToForm';
 import { errorCreator } from 'capture-core-utils';
 import { actionCreator } from '../../../actions/actions.utils';
 import { addFormData } from '../../D2Form/actions/form.actions';
-import { getRulesActionsOnLoad, getRulesActionsOnLoadForSingleNewEvent } from '../../../rulesEngineActionsCreator/rulesEngineActionsCreatorForEvent';
+import { getRulesActionsOnLoad, getRulesActionsOnLoadForSingleNewEvent } from '../../../rules/actionsCreator';
```

Comment:
this file looks like is not being used since this imports dont exist and it didnt also before this PR.

Would you like to introduce me to the history of this @JoakimSM?

Reply:
Gonna go on and delete this one while doing the lgtm fixes :) I will follow up on this.